test/extended/util/volumes.go

```diff
@@ -0,0 +1,13 @@
+package util
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+)
+
+func CleanUp(oc *CLI) {
```

Comment:
if you're moving stuff around, why not move the SetupHostPathVolumes in here too?

Reply:
Updated in current push